Sources/ApodiniGRPC/Server.swift

```diff
@@ -0,0 +1,289 @@
+//
+// This source file is part of the Apodini open source project
+//
+// SPDX-FileCopyrightText: 2019-2021 Paul Schmiedmayer and the Apodini project authors (see CONTRIBUTORS.md) <paul.schmiedmayer@tum.de>
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Apodini
+import ApodiniExtension
+import NIO
+import NIOHPACK
+import Foundation
+import ApodiniUtils
+import ApodiniNetworking
+import ProtobufferCoding
+
+
+struct GRPCServerError: Swift.Error {
+    let message: String
+}
+
+
+/// The gRPC server manages a set of gRPC services (as well as their methods),
+/// and implements the logic for handling incoming requests to these methods.
+class GRPCServer {
```

Comment:
Would be nice if we could rename this file to `GRPCServer.swift` took me a bit to find it 🙈

Reply:
I mean, we can of course rename it, though if i use the cmd-shift-option thing to search for either "grpcserver" or just "server", it always is within the first 5 results...